docs/run_test/understand_results.md

```diff
@@ -0,0 +1,46 @@
+# Understand test results
```

Comment:
The content is more about how to write those log and error messages, not about analyzing results.

Reply:
We may can ask user let us know, if some error messages are hard to understand.

Reply:
I like this info, myself. Also, quick note: there are *lots* of grammar/composition issues on our code-thrown messages themselves. I hope Wenyi will have the time/will to work on that too :)

Reply:
I used content from how to write log and error messages, but I think it's also helpful to write such a guide for test runners to understand or perform a preliminary analyze on test results. I hope I would get a chance looking at code-thrown messages after restructuring all the docs.